webextension/background.js

```diff
@@ -105,6 +161,11 @@ const Config = (function() {
         for (const [name, value] of Object.entries(otherPrefs)) {
           this[`_${name}`] = value;
         }
+
+        // If a valid UI variant has not otherwise been chosen yet, select one now.
+        if (!this._uiVariant) {
+          this._selectRandomUIVariant();
```

Comment:
I'm not sure we're going to need this... if the test plan requires QA to manually enter the variant, and the addon chooses the variation for a user, what's the use case? I'm afraid if real users enter into this, our assumptions about populations  and data are basically wrong.

Reply:
This is just where the addon randomly chooses a variation for now, as it has to pick one, right?

(The actual selection will presumably end up being handled by the relevant Shield `browser.study` API later, but for now I just put in placeholder code that uses `Math.random()`).